standalone/prouter/src/reseeder.rs

```diff
@@ -0,0 +1,122 @@
+use anyhow::{anyhow, Result};
+use byteorder::{BigEndian, WriteBytesExt};
+use glob::glob;
+use rand::seq::SliceRandom;
+use std::fs;
+use std::fs::read;
+use std::io::{BufReader, Read, Write};
+use std::path::{Path, PathBuf};
+use std::time::{SystemTime, UNIX_EPOCH};
+
+#[derive(Debug, Clone)]
+pub struct SU3File {
+    pub signer_id: String,
+    signer_id_length: u8,
+    signature_type: u16,
+    signature_length: u16,
+    version_length: u8,
+    file_type: u8,
+    content_type: u8,
+    content: Vec<u8>,
+    content_length: u64,
+    version: String,
+}
+
+impl SU3File {
+    pub fn new(signer_id: &str) -> Result<SU3File> {
+        Ok(SU3File {
+            signer_id: signer_id.to_string(),
+            signer_id_length: signer_id.chars().count() as u8,
+            signature_type: 6, // 0x0006
+            signature_length: 512,
+            version_length: 16, // 0x10
+            file_type: 0,       // 0x00
+            content_type: 3,    // 0x03
```

Comment:
Seems like the comments here are meaningless. If the hex version is preferred, it could be written as `version_length: 0x10`.